lib/datagrid/core.rb

```diff
@@ -52,6 +52,7 @@ def self.included(base)
         class_attribute :datagrid_attributes, instance_writer: false, default: []
         class_attribute :dynamic_block, instance_writer: false
         class_attribute :forbidden_attributes_protection, instance_writer: false, default: false
+        class_attribute :default_filter_options, default: {}
```

Comment:
Add the docs for the new setting and update existing ones on columns:

https://rubydoc.info/gems/datagrid/Datagrid/Columns#default_column_options-instance_method

Reply:
Updated the docs and added some more specs where felt appropriate. Rails 7.0 test suite keeps failing tho - looks to be something more global than my changes.

Reply:
I fixed failing tests in `main`.